readme.md

```diff
@@ -7,5 +7,37 @@ Halcyon is a Home Assistant app for Linux built in rust.
 ### Prerequisites
 
 * Rust
+* libssl-dev (or equivalent package for your distro)
 
 To build Halcyon, run `cargo build`.
+
+Only the setup command is working at the moment
+To run it do the following
+`cargo build` then run the halcyon executable
+`./target/debug/halcyon setup`
```

Comment:
I'd say either we should have people use `cargo run`, or get them to compile it in release mode and use the binary itself.